src/estimation/include/iDynTree/Estimation/ExtendedKalmanFilter.h

```diff
@@ -161,6 +163,21 @@ namespace iDynTree
          */
         void ekfReset();
 
+        /**
+         * @brief Resets the filter flags, initializes and resizes internal buffers of the filter, and
+         *             sets initial state, initial state covariance, and system noise and measurement noise covariance matrices
+         * @note this method is particularly useful while working with hybrid systems,
+         *             where the size of the system state or the measurements keep evolving with time
+         *
+         */
+        bool ekfReset(const size_t& state_size,
+                                 const size_t& input_size,
+                                 const size_t& output_size,
+                                 const iDynTree::Span<double>& x0,
+                                 const iDynTree::Span<double>& P0,
+                                 const iDynTree::Span<double>& Q,
+                                 const iDynTree::Span<double>& R);
```

Comment:
Q and R are typically matrices, what is the expected size of the Span<double> to represent them? Is the ordering column major or row major?